### PR TITLE
[TDL-71, TDL-72, TDL-73, TDL-74] New invoice flow

### PR DESCRIPTION
**Pull request by ardier16**
Waiting for SDK release

### CODE REVIEW

File: src/vue/modules/loyalty-points/create-invoice-form/components/invoice-confirmation.vue [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="create-invoice-form__payment">
```
Comment (max-nes) [inline src/vue/modules/loyalty-points/create-invoice-form/components/invoice-confirmation.vue]: invalid class

File: src/vue/modules/loyalty-points/create-invoice-form/components/invoice-confirmation.vue [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    <invoice-viewer :invoice="invoice" />
+
+    <button
+      v-ripple
+      type="button"
+      class="create-invoice-form__close-btn app__button-raised"
```
Comment (max-nes) [inline src/vue/modules/loyalty-points/create-invoice-form/components/invoice-confirmation.vue]: invalid class

File: src/vue/modules/loyalty-points/create-invoice-form/components/invoice-confirmation.vue (hunk 1 above)
Comment (ardier16) [reply src/vue/modules/loyalty-points/create-invoice-form/components/invoice-confirmation.vue]: Renamed

File: src/vue/modules/loyalty-points/create-invoice-form/components/invoice-confirmation.vue (hunk 2 above)
Comment (ardier16) [reply src/vue/modules/loyalty-points/create-invoice-form/components/invoice-confirmation.vue]: Renamed

**Pull request merged**